config/essi_config.example.yml

```diff
@@ -31,6 +31,9 @@ default: &default
       user_name: 'example@gmail.com'
       password: 'app_password'
       authentication: 'plain'
+      raise_delivery_errors: 'false'
+      enable_starttls_auto: 'true'
```

Comment:
@randalldfloyd These are probably functionally equivalent but changing them from strings to raw true/false values would be more consistent with other config values in this section

Reply:
@aploshay Yeah didn't even catch that I had done that.